packages/api/src/controllers/taskController.js

```diff
@@ -36,9 +36,15 @@ const taskController = {
           return res.status(400).send('Task has already been completed or abandoned');
         }
 
+        if(!adminRoles.includes(req.role) && checkTaskStatus.assignee_user_id != req.user.user_id && checkTaskStatus.assignee_user_id !== null){
+          return res.status(400).send('Farm workers are not allowed to reassign a task assigned to another worker');
```

Comment:
```suggestion
          return res.status(403).send('Farm workers are not allowed to reassign a task assigned to another worker');
```
I believe a 403 error code makes more sense here because it is an unauthorized error https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses